### ISSUE TITLE
Fix the 'out of command' rule, and thereby solve the _Infiltrators_ 'scouting' conundrum...?

### ISSUE THREAD

**Issue by digi-brain** [GameBrains/er-core#50]
We've been stuck on how to make full sense of the 'scouting'<sup>(1)</sup> aspect of the _Infiltrators_ ability — but I think that I may have a way forward.

> <sup>(1)</sup> Where 'scouting' refers to the part of the _Infiltrators_ ability that comes into force when the unit is >30cm <=60 cm away from its detachment HQ.

**TL;DR**: I believe that the problem is not so much to do with the _Infiltrators_ ability rules text as with incompleteness of the 'out of command' rules text. The out of command rule actually has some inherent problems which it'd be good to fix anyway, **independent** of the _Infiltrators_ ability… but what's interesting is that in doing that, I believe that the 'scouting' part of the _Infiltrators_ ability will make sense.

**The long version:**

First, let's remind ourselves of the different specific 'scouting'-related clauses that we have in the original texts:

* **Clause 1**: [Infiltrators](https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/SpecialistsAndSpecialAbilities/Infiltrators.htm): "Infiltrators units count as being within command distance if they are within 60cm of their detachment HQ."
* **Clause 2**: [Infiltrators](https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/SpecialistsAndSpecialAbilities/Infiltrators.htm): "An Infiltrators unit which is more than 30cm from its HQ unit is still destroyed if it is broken in close combat or a firefight."
* **Clause 3**: [HQs](https://thehobby.zone/resources/e40k-compendium/Content/CoreRules/HQs.htm): "Only units within 30cm of their HQ unit at the start of a phase are 'in command' so they can move and/or shoot in that phase."
* **Clause 4**: [FAQ: Out of Command units](https://thehobby.zone/resources/e40k-compendium/Content/More/HouseRulesFrequentlyAskedQuestionsAndOtherNonsense/FAQ_OutOfCommandUnits.htm): "Don't place Blast markers on out of command units because it has weird effects on army morale (the units usually get killed, leading to the Blast markers being deducted from the army morale level immediately). "
* **Clause 5**: [Broken detachments](https://thehobby.zone/resources/e40k-compendium/Content/CoreRules/BrokenDetachments.htm): "Units that are 'out of command' (more than 30cm from their detachment HQ unit) are not affected if their parent detachment is broken (so they don't have to retreat). Conversely, if a unit that is out of command is broken, then it is destroyed with no other effects on the parent detachment."

---

Now, note that **clause (2) alludes to clause (5)**, which means that **clause (5) is in effect for scouting _Infiltrators_** just as it is for other units that are  >30cm from their HQ. In other words, **scouting Infiltrators units are definitely out of command as a baseline**, and should behave as such in all cases where we don't have specific exceptions that apply to make the "60 cm command distance" bit meaningful.

I can't see any other way to interpret this<sup>(2)</sup> -- but we overlooked or lost sight of it in the rabbit holes we went down in discussions in the Facebook group.

> <sup>(2)</sup> It also seems more parsimonious to me than the idea that they are in command as a baseline -- otherwise we have the oddity whereby we have to have an exception to the exception just to get back to where we started, to define when scouting units do in fact follow parts of the out of command rule.

Further evidence in support of this conclusion:

1. Scouting _Infiltrators_ units only "**count as** being within command distance" — in other words, they are **not actually** in command. We just treat them as in command for some specific (badly explained) purposes that pertain to their ability to respond to commands (orders). Ergo, outside of these specific purposes, they are actually out of command.
2. Either way, "within command distance" is an odd turn of phrase — but it seems more parsimonious to me to take it to mean "able to act on orders **as if** they were within the usual 30 cm radius and therefore in command" rather than "they are **actually** in command even though they are >30cm <=60 cm away".
3. It seems to me that the allusion of clause (2) to clause (5), and use of the word "still", only makes sense if scouting units are out of command as a baseline, like any other unit at the same distance from its HQ unit.

In essence, the specific purposes for which scouting units **count as** in command pertain to the ability to act on orders, in contrast to the usual limited set of reactions that apply to other out of command units. That is, the controlling player can move and/or shoot with scouting units in response to their detachment's orders **as if** they are in command, even though they are not.

I _think_ that up to this point, the above conclusions all work fine as-is. The tricky bit is how we deal with shooting and assaults that target scouting _Infiltrators_ units -- and _that_ boils down not to the _Infiltrators_ rule itself but rather to how targeting of **any** out of command units should work.

---

Further note that clause (5) clearly treats out of command units and the rest of the parent detachment as **separate entities** when it comes to both Blast markers and the Broken condition. Given the above, this separateness is important not just for targeting of out of command units in general but also for targeting of scouting _Infiltrators_ units, to make sense of how scouting _Infiltrators_ can actually work.

Clause (4) makes states that we don't place Blast markers on out of command units — and, given the separateness implied by clause (5), I think it is parsimonious to conclude that clause (4) means that we don't place those Blast markers **at all**. That is, however targeting of out of command units should work exactly, we do not shift Blast markers that they would otherwise receive onto the parent detachment instead, because for these purposes it is effectively a separate entity.

---

If you accept all the above, then the real uncertainty is not with the scouting aspect of _Infiltrators_ ability, but rather with our understanding of how to resolve shooting and assaults with out of command units in general.

Altogether, I think we have to conclude that when shooting at or assaulting out of command units (which includes scouting units), we must target/resolve them separately from the rest of their parent detachment, to make sense of clause (5).

What does that mean in practice? I'm not entirely sure, this is what we need to figure out. But for starters I think that it might have to go something like this:

* When you wish to target out of command units with your shooting, target those from a particular detachment as though they form their own separate detachment. But don't place Blast markers on these units, nor on their parent detachment.
* When you also want to target units in the rest of the parent detachment, then you must use the rules for splitting your shooting between multiple detachments — because the parent detachment and its out of command units are effectively separate for these purposes. Don't place Blast markers on the out of commands units at all, but place Blast markers on their parent detachment as you usually would for the part of the split fire that is actually directed at them.
* Similarly, when you assault out of command units, treat those from a given parent detachment as though they form their own separate detachment from that parent detachment. Further, I guess it'd make most sense to ignore any that aren't actually engaged (base contact or supporting, that is within 15 cm), in the spirit of clause (5) — that is, if you win the assault then apply the latter part of clause (5) only to the out of command units that were actually engaged. Otherwise it gets a bit silly when some other out of command units from that same parent detachment are half-way across the table somewhere.

In effect, we treat each group of targeted out of command units<sup>(3)</sup>  from a given enemy parent detachment (or perhaps any enemy parent detachment?) as an **instantaneous** separate detachment for the duration of that particular bit of shooting, close combat or firefight.

> <sup>(3)</sup> Where the 'group of targeted out of command units' is those which are in range/line of fire for shooting, or that are 'engaged' (base contact or within 15 cm) in close combat or firefight. It'd also be helpful to define 'engaged' clearly in the assault rules anyway, so this is just another application of an existing principle.

---

What say you @IJW-Wartrader

**Comment by IJW-Wartrader** [comment]
I'm pretty sure we went through every one of those points before. 😜

Hmm...

> Further evidence in support of this conclusion:
> 
> 1. Scouting _Infiltrators_ units only "**count as** being within command distance" — in other words, they are **not actually** in command. We just treat them as in command for some specific (badly explained) purposes that pertain to their ability to respond to commands (orders). Ergo, outside of these specific purposes, they are actually out of command.
> 2. Either way, "within command distance" is an odd turn of phrase — but it seems more parsimonious to me to take it to mean "able to act on orders **as if** they were within the usual 30 cm radius and therefore in command" rather than "they are **actually** in command even though they are >30cm <=60 cm away".
> 3. It seems to me that the allusion of clause (2) to clause (5), and use of the word "still", only makes sense if scouting units are out of command as a baseline, like any other unit at the same distance from its HQ unit.

1. I think that saying "**count as**" can be treated as "**not actually**" is opening up a huge can of worms, given how liberally the phrase is used in the original rules text. For example, if a piece of terrain "counts as dangerous terrain", what are the implications of saying that it's not actually dangerous terrain? Units will still be taking dangerous terrain tests. If war engines "count as having the special ability Stubborn", what are the implications of them not actually having Stubborn? They will still get to re-roll the Leadership test. Repeat for Pulsars counting as a single Super Heavy Weapon, etc. etc.
2. Yes and no. It's not like the rest of the rules are consistent about phrasing. :-( "In command" is only used in the HQ section and the Imperial Guard Q&A (where it also swaps between "in command" and "within command"). And a non-staff tactics article.
3. Within context, "still" appears to be phrased as an exception. Paraphrased very loosely, my reading of the paragraph is 'Infiltrators units count as X, but in Y situation they still suffer Z'. 

To bring up some points from previous discussions:

If we default to Infiltrators being in command, we only need to specify what happens if they are broken. 

If we default to Infiltrators being out of command, we have to specify _every_ situation in which they count as being in command. Such as:
- Movement.
- Shooting.
- Assaulting. Specifically, why the Infiltrators are part of an assaulting detachment for movement and working out Assault or Firepower totals, but not for breaking. Do they count for allocating hits to them or from them? Do they count as units in base contact for how many Blast markers can be inflicted on the loser? Does an Infiltrators unit in base contact with the enemy allow non-Infiltrators to add +1 in support, and vice-versa?
- Rallying (which can happen if the detachment breaks and then the Infiltrators end up outside 30 cm).

And the big issue, which is that on the tabletop, completely disposable Infiltrators are horribly _horribly_ broken for only being part of a 1pt ability, and can be nastily abused. 

Especially when in transports, such as Scouts in Razorbacks or Drop Pods, Kommandos in Trukks, Eldar Scouts in Falcons or Wave Serpents (although that gets more expensive), Chaos Marine Veterans in Rhinos, etc. March them up the table with a move/move+disembark/move. The transports and HQ can all stay out of sight/move back out of sight while the Infiltrators block a big swathe of the table. Your opponent can't see the non-Infiltrators without Artillery or very long-range Skimmers, so it's almost impossible for your opponent to inflict Blast markers or to break anything in an assault.


> * When you wish to target out of command units with your shooting, target those from a particular detachment as though they form their own separate detachment. But don't place Blast markers on these units, nor on their parent detachment.

In practice, we've found the easiest and most consistent option is to treat all OoC units in range to be one 'target detachment' for shooting. With no Blast markers being placed. 

Treating all the OoC units from a detachment as a separate target from ones from another detachment is another bit that's open to abuse - a few small detachments push forwards and spread out their Infiltrators (or just regular units moving to be OoC) in an intermingled pattern, meaning that each unit ends up being a completely separate target.

**Comment by digi-brain** [comment]
> I'm pretty sure we went through every one of those points before. 😜

Mmm... possibly, but looking at it recently I feel like we must've lost sight of the significance in how clause (2) alludes to clause (5).

It's entirely possible that my memory is not serving me well, so I won't labour the point **too** long if you remain convinced that the impasse remains as intractable as ever because my 'out of command' hypothesis can't work any better than the 'in command' hypothesis.

---

> I think that saying "**count as**" can be treated as "**not actually**" is opening up a huge can of worms, given how liberally the phrase is used in the original rules text. For example, if a piece of terrain "counts as dangerous terrain", what are the implications of saying that it's not actually dangerous terrain? Units will still be taking dangerous terrain tests. If war engines "count as having the special ability Stubborn", what are the implications of them not actually having Stubborn? They will still get to re-roll the Leadership test. Repeat for Pulsars counting as a single Super Heavy Weapon, etc. etc.

Perhaps I overstated: I don't mean that we should automatically interpret all instances of "count as" to equate to a definite "not actually". But I take your point on how 'count as' terminology is inconclusive given the way that it is used elsewhere.

We can debate the extent to which the 'count as' phrasing might or might not be supporting evidence — but at minimum it leaves the "not actually" door open wide enough that it doesn't inherently contradict my 'out of command' hypothesis.

---

But let's put the secondary stuff aside for a moment and focus on my fundamental point:

It seems to me that interpretation of the clause (2) reference to clause (5) is critical. And here we disagree on the meaning of the words.

Let's also put aside the wider analysis of the further practicality of my hypothesis for a moment, and focus on interpretation of these clauses as written.

> * **Clause 2**: [Infiltrators](https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/SpecialistsAndSpecialAbilities/Infiltrators.htm): "An Infiltrators unit which is more than 30cm from its HQ unit is still destroyed if it is broken in close combat or a firefight."
> * **Clause 5**: [Broken detachments](https://thehobby.zone/resources/e40k-compendium/Content/CoreRules/BrokenDetachments.htm): "Units that are 'out of command' (more than 30cm from their detachment HQ unit) are not affected if their parent detachment is broken (so they don't have to retreat). Conversely, if a unit that is out of command is broken, then it is destroyed with no other effects on the parent detachment."

This is the crux of it I think: Logically, it seems to me that the word "still" is **totally** redundant if it does not refer to an **existing** state — you can't "still" be anything that you weren't already. Though oblique, it seems that clause (2) can only be a reference to clause (5) — and the existing state in that clause is 'out of command'. This use of the word "still" strongly implies that the writer took for granted that scouting _Infiltrators_ units are out of command by default.

**Ergo, scouting units are out of command, as a baseline.**

Otherwise, you have to explain away this use of the word "still" as a completely arbitrary mistake, and explain why, even if they didn't think about it enough, the writer didn't naturally default to a phrase that fit more cleanly with the idea that these units are in command, as a baseline. I mean, it would've been quite natural to write something like:

* "However, an Infiltrators unit which is more than 30cm from its HQ unit is destroyed if it is broken in close combat or a firefight, much like a unit that is out of command."

...but they didn't. Yes, it's not conclusive proof given all the proclivities for dodgy writing. But again, I really think it's more parsimonious not to have to explain this away and rather to accept that "still" is intentional and implies that the writer took the out of command state for granted as the baseline for scouting units.

---

Secondary to my fundamental point above, but related:

> 3\. Within context, "still" appears to be phrased as an exception. Paraphrased very loosely, my reading of the paragraph is 'Infiltrators units count as X, but in Y situation they still suffer Z'.

That nonetheless reads to me as '[Scouting] Infiltrators units count as X [in exception to the usual rules for the state that exists], but in Y situation they still suffer Z [as an exception to the exception we just made, to get back to the state that already exists]'.

That's not impossible, but you're usually the one pointing out that a given interpretation demands an exception to an exception, which is not parsimonious. That's fair enough and is often a strong point. However, in this case, you seem to be arguing for scouting units to be 'in command' as a baseline — which is an exception to the normal <=30cm requirement for a unit to be in command rather than out of command — and then also to treat the reference from clause (2) to clause (5) as a further exception that **countermands** an aspect of the exception we just made. This seems very odd to me.

---

> In practice, we've found the easiest and most consistent option is to treat all OoC units in range to be one 'target detachment' for shooting. With no Blast markers being placed.
>
> Treating all the OoC units from a detachment as a separate target from ones from another detachment is another bit that's open to abuse - a few small detachments push forwards and spread out their Infiltrators (or just regular units moving to be OoC) in an intermingled pattern, meaning that each unit ends up being a completely separate target.

So, at this point we agree that we neither place Blast markers on OoC units and nor do we shift those that would've been placed onto the parent detachment instead. We also seem to agree that we treat all in range/line of fire or 'engaged' OoC units as a distinct instantaneous detachment while we resolve the shooting/assault. (I wasn't entirely sure on the detail, but my query near the end -- "(or perhaps any enemy parent detachment?)" -- corresponds to what you're saying here about potential for abuse.)

---

> If we default to [scouting] Infiltrators being in command, we only need to specify what happens if they are broken.

Maybe, but we've clearly reached an impasse with that hypothesis where it basically doesn't do anything interesting like we expected. We've reach a position of "this will have to do for want of a better idea". That might indeed have to do, but I wonder if my hypothesis can work out.

> If we default to Infiltrators being out of command, we have to specify _every_ situation in which they count as being in command.
> Such as:
> 
>     * Movement.
> 
>     * Shooting.
> 
>     * Assaulting. Specifically, why the Infiltrators are part of an assaulting detachment for movement and working out Assault or Firepower totals, but not for breaking. Do they count for allocating hits to them or from them? Do they count as units in base contact for how many Blast markers can be inflicted on the loser? Does an Infiltrators unit in base contact with the enemy allow non-Infiltrators to add +1 in support, and vice-versa?
> 
>     * Rallying (which can happen if the detachment breaks and then the Infiltrators end up outside 30 cm).

Yes and no. (And I admit that this is where I get a bit hazy and haven't worked it all out.)

The most fundamental thing about a unit being in command is that you can use it in accordance with orders -- in contrast to out of command units with which you can only perform some limited responses:

* [HQs](https://thehobby.zone/resources/e40k-compendium/Content/CoreRules/HQs.htm): "Only units within 30cm of their HQ unit at the start of a phase are 'in command' so they can move and/or shoot in that phase. So, for example, units that start more than 30cm from their HQ can't move in the Movement phase or Assault phases, or shoot in the Shooting phase."
* [HQs](https://thehobby.zone/resources/e40k-compendium/Content/CoreRules/HQs.htm): "You do not need to be 'in command' to carry out any of the following actions: snap-fire, fight or lend support in an assault, shoot in a firefight. These actions are explained in detail later in the rules."

So, under my 'OoC by default' hypothesis I think we only really need to specify that scouting _Infiltrators_ units count as in command for the purposes of acting on orders. This is not how I write it, but in effect we get this -- to parallel the first bullet above that applies to regular in command units, but with adjustments specifically for scouting units:

* "Scouting units -- those _Infiltrators_ units that are >30cm <=60cm from their HQ unit at the start of a phase -- count as 'in command' so they can move and/or shoot in that phase." Of course, _Infiltrators_ units that start the phase at more than 60cm from their HQ are not scouting, so they are fully out of command with no 'treat as in command' exceptions; and those that are within 30cm at the start of the phase operate like any other truly in command unit. (Yes, we may have some weird edge cases to figure out when the 'scouting' state changes, and I don't have these in mind as you may have — but I'm hopeful that these will prove to have sensible solutions.)

That is, we know that we can move, shoot and assault with 'OoC by default, but count as in command for orders' scouting units, like we can with regular units that are truly in command. 

>     * Assaulting. Specifically, why the Infiltrators are part of an assaulting detachment for movement and working out Assault or Firepower totals, but not for breaking. Do they count for allocating hits to them or from them? Do they count as units in base contact for how many Blast markers can be inflicted on the loser? Does an Infiltrators unit in base contact with the enemy allow non-Infiltrators to add +1 in support, and vice-versa?

As I've just outlined, scouting units count as in command for the sake of acting on orders, so can can move, shoot and assault much like other units in their parent detachment.

> Do they count for allocating hits to them or from them?

Allocating hits in an assault seems easy — whether its a single detachment or multiple detachments makes no difference to hit allocation, does it? After all, "Where two or more detachments are in contact with one or more enemy detachments, resolve the fight as a single large combat." ([Multiple close combats](https://thehobby.zone/resources/e40k-compendium/Content/CoreRules/MultipleCloseCombats.htm))

So yes, you can allocate hits to and from scouting units. Besides, we've already seen that the rules state that even any old out of command unit can "fight or lend support in an assault" if they find theirselves in one.

> Do they count as units in base contact for how many Blast markers can be inflicted on the loser?

Unless I'm missing your point, this is a question to resolve for all out of command units and not not particular to my hypothesis for scouting _Infiltrators_ units, isn't it?

But I imagine, given that even out of command units can "fight or lend support in an assault", it works as normal. It's just that (a) the 'instantaneous' detachment we discussed is in effect for all involved OoC units (including scouting units), and per clause (4)  this instananeous detachment doesn't take any of the Blast markers and the units are destroyed so don't get to make a retreat move, and (b) if involved, the parent detachment gets its own share as normal for any separate detachment, limited by the number of units it has in contact **excluding** the scouting units because they're currently treated as part of the separate instantaneous detachment.

Where it might get a bit odd is remembering which specific units were scouting at the start of the Assault phase. But that could also be true for regular OoC units in some cases, where the HQ moves back into range during the Assault phase -- so again it's a more general OoC issue, not an issue inherent to my hypothesis.

> Does a [scouting] Infiltrators unit in base contact with the enemy allow non-Infiltrators to add +1 in support, and vice-versa?

Again, it's a general OoC issue, not an issue inherent to my hypothesis. I'd say yes, given that the rules say that even any old OoC unit can "fight or lend support in an assault".

> * Rallying (which can happen if the detachment breaks and then the Infiltrators end up outside 30 cm).

If I understand you right: The _Infiltrators_ units were within 30 cm of their HQ at the start of the Assault phase and therefore fully part of their detachment in the normal way, and then the detachment lost the assault an made it's retreat move. Either due to the movement or due to loss and of HQ unit and different position of the replacement, some _Infiltrators_ units end the Retreat move in a scouting position (>30cm <=60cm from their HQ unit).

In my hypothesis, these scouting units are now out of command by default, and you're thinking "now what?"

Okay, how is this different to the same situation but with regular units that are >30cm from their HQ after a Retreat move? What do we do in that case, and why can't the same apply for _Infiltrators_ units that are in a scouting position after a Retreat move? Isn't this another general OoC edge case issue to fix rather than one inherent to my hypothesis?

---

> And the big issue, which is that on the tabletop, completely disposable Infiltrators are horribly horribly broken for only being part of a 1pt ability, and can be nastily abused.

Okay, I don't have my head wrapped around this one, but assuming everything else I've said makes sense then hopefully this goes away or it's a problem we can solve with some more thought.

---

Phew! 

If you've taken all this on board and you're still adamant that there's no workable sense in my hypothesis, then I'll drop it.

If you see some workable sense but you're on the fence because some details seem like blockers, it may be best if we discuss in a video call as that's likely to be much quicker to thrash out one way or another. This one written reply has taken up most of my Sunday on top of the time to write out the first post to start the thread, and I'm sure it goes much the same for you -- we should short-cut it if we can. 😂

**Comment by IJW-Wartrader** [comment]
> Phew!

If you've taken all this on board and you're still adamant that there's no workable sense in my hypothesis, then I'll drop it.

If you see some workable sense but you're on the fence because some details seem like blockers, it may be best if we discuss in a video call as that's likely to be much quicker to thrash out one way or another. This one written reply has taken up most of my Sunday on top of the time to write out the first post to start the thread, and I'm sure it goes much the same for you -- we should short-cut it if we can. 😂

I think there might be some workable sense, but I also think getting to that sense might involve a huge amount of effort, and still wouldn't solve any of the balance issues with Infiltrators defaulting to being OoC. 

In any case, the start of this week is pretty busy but I could do something on Wednesday or Thursday.


Some more points I'm not 100% sure I've mentioned before...

> **Ergo, scouting units are out of command, as a baseline.**
> 
> Otherwise, you have to explain away this use of the word "still" as a completely arbitrary mistake, and explain why, even if they didn't think about it enough, the writer didn't naturally default to a phrase that fit more cleanly with the idea that these units are in command, as a baseline. I mean, it would've been quite natural to write something like:
> 
> * "However, an Infiltrators unit which is more than 30cm from its HQ unit is destroyed if it is broken in close combat or a firefight, much like a unit that is out of command."
> 
> ...but they didn't. Yes, it's not conclusive proof given all the proclivities for dodgy writing. But again, I really think it's more parsimonious not to have to explain this away and rather to accept that "still" is intentional and implies that the writer took the out of command state for granted as the baseline for scouting units.

I'm more concerned that, if the scouting Infiltrators default to being OoC, the text only mentions being broken in an assault, without covering 'what happens if I shoot the unit(s)?'. 

With shooting being a more common action than assaulting, the results of shooting would be a higher priority to mention.

This:

> * "However, an Infiltrators unit which is more than 30cm from its HQ unit is destroyed if it is broken in close combat or a firefight, much like a unit that is out of command."

Would need to be more like:

* "However, any Infiltrators units which are more than 30cm from their HQ unit are treated as a separate detachment when shot at and never suffer Blast markers, and are destroyed if they are broken in close combat or a firefight, much like units that are out of command."

Anyway, to sum up my position from a different direction, I do think the mention of still being destroyed if broken was dodgy writing. But not in terms of whether the scouting Inflitrators are actually OoC or not, rather in terms of being an incomplete rule or the result of a rule that got changed in testing and the text was only partially updated. It reads like something that was half-edited to change how the rule worked, but the editing was never finished and old text got left in. 🙁

**Comment by IJW-Wartrader** [comment]
P.S. I do agree that the general rules for OoC units need some work.

**Comment by digi-brain** [comment]
Wednesday or Thursday for a video call works for me, if you get a window.

---

> still wouldn't solve any of the balance issues with Infiltrators defaulting to being OoC.

I'm sure you've mentioned it, but I'm not joining the dots right now to recall what you mean by 'balance issues' in this context. I'm sure we'll get to that.

---

> I'm more concerned that, if the scouting Infiltrators default to being OoC, the text only mentions being broken in an assault, without covering 'what happens if I shoot the unit(s)?'.

The root problem here is with the OoC rules, isn't it? Whether or not my hypothesis has merit, "what happens when you shoot at OoC units" is still an issue. That's why I gave this issue ticket the title I did — it seems to me that most of the problems with the 'scouting units are OoC by default' due to the OoC rules, not the hypothesis itself. You'll notice that in my last reply there are several times I've noted something along the lines of "Again, this is a problem inherent to the OoC rules, not my hypothesis..."

---

> This:
> 
> > * "However, an Infiltrators unit which is more than 30cm from its HQ unit is destroyed if it is broken in close combat or a firefight, much like a unit that is out of command."
> 
> Would need to be more like:
> 
>     * "However, any Infiltrators units which are more than 30cm from their HQ unit are treated as a separate detachment when shot at and never suffer Blast markers, and are destroyed if they are broken in close combat or a firefight, much like units that are out of command."

I was making a different point to the one you've latched onto.

The hypothetical text I gave in that bullet **deliberately** shows writing that is still crappy and incomplete, very similar to the original.

The point was to show that if the writer **did not** assume OoC as the default and **did not** mean to deliberately recall this in the way that the word "still" implies, then it's a very odd choice that I wouldn't expect to flow naturally from the writer's mind... and I gave an example of a more natural phrase that one might expect if the writer actually had the 'in command by default' interpretation in mind (I removed "still" which wouldn't be very natural for that assumption, and added "much like..." instead).

Again it's far from conclusive evidence, and we can never truly know and can cook up other theories about how the text might've been mucked about with — but I'm trying to work with a picture of circumstantial evidence that adds up to what I think is a more parsimonious interpretation as far as the actual written words are concerned.

**Comment by digi-brain** [comment]
To summarise key conclusions we arrived at in our video call today:

Next steps:

1. Finalise and publish #47. This gives us something that is at least workable and simple, both for now and for the long term in the event that the 'OoC scouting Infiltrators' hypothesis I've proposed here doesn't eventually result in a better solution.
2. Improve the out of command rules, which for many situations are unclear or even incomplete. This is desirable anyway, but should also fix the root cause of most of the difficulties with this hypothesis.
3. Test the hypothesis with the improved OoC command rules, and work on a new version of the relevant _Infiltrators_ rules text. If and when the proves fruitful, publish the new version to replace the interpretation and compromise we made under #47.

Points that relate to specfiic difficulties that OoC fixes alone won't resolve:

* We suspect that the scouting rules should apply only while the parent detachment is not subject to the Broken condition. This mean that while the parent unit is in Broken condition, any _Infiltrators_ units that are in a position that would've counted for scouting revert to the standard OoC rules instead, just like any other unit.
